### PR TITLE
Update Get-SPOGeoStorageQuota.md

### PR DESCRIPTION
**Pull request by PowershellScripts**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOGeoStorageQuota.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Get-SPOGeoStorageQuota [-AllLocations] [<CommonParameters>]
 
 This cmdlet shows the storage on the current location or all locations in a multi-geo tenant.
 
-This Cmdlet requires a connection to a multi-geo tenant to run correctly. You must be a SharePoint Online global Administrator to get the storage on current location or all locations in a multi-geo SPO tenant.
+This cmdlet requires a connection to a multi-geo tenant to run correctly. You must be a SharePoint Online global Administrator to get the storage on current location or all locations in a multi-geo SPO tenant.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOGeoStorageQuota.md:29]:
```suggestion
This cmdlet requires a connection to a multi-geo tenant to run correctly. You must be a SharePoint Online global administrator to get the storage on the current location or all locations in a multi-geo SPO tenant.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - @PowershellScripts kindly commit the minor additional suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOGeoStorageQuota.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ Get-SPOGeoStorageQuota [-AllLocations] [<CommonParameters>]
 
 This cmdlet shows the storage on the current location or all locations in a multi-geo tenant.
 
-This Cmdlet requires a connection to a multi-geo tenant to run correctly. You must be a SharePoint Online global Administrator to get the storage on current location or all locations in a multi-geo SPO tenant.
+This cmdlet requires a connection to a multi-geo tenant to run correctly. You must be a SharePoint Online global administrator to get the storage on the current location or all locations in a multi-geo SPO tenant.
```
Comment (tseward) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOGeoStorageQuota.md:29]:
```suggestion
This cmdlet requires a connection to a multi-geo tenant to run correctly. You must be a SharePoint Admin or Global Administrator to get the storage on the current location or all locations in a multi-geo SPO tenant.

**Review by tseward — CHANGES_REQUESTED** [review]

**Comment by yogkumgit** [comment]
@dariomws @get-itips

**Review by dariomws — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOGeoStorageQuota.md [hunk 3]
```diff
@@ -26,7 +26,7 @@ Get-SPOGeoStorageQuota [-AllLocations] [<CommonParameters>]
 
 This cmdlet shows the storage on the current location or all locations in a multi-geo tenant.
 
-This Cmdlet requires a connection to a multi-geo tenant to run correctly. You must be a SharePoint Online global Administrator to get the storage on current location or all locations in a multi-geo SPO tenant.
+This cmdlet requires a connection to a multi-geo tenant to run correctly. You must be a SharePoint Admin or Global Administrator to get the storage on the current location or all locations in a multi-geo SPO tenant.
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOGeoStorageQuota.md:29]:
```suggestion
This cmdlet requires a connection to a multi-geo tenant to run correctly. You must be a SharePoint Online Administrator or Global Administrator to get the storage on the current location or all locations in a multi-geo SPO tenant.
```

**Comment by yogkumgit** [comment]
@get-itips

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**